.github/workflows/assign-class-label.yaml

```diff
@@ -0,0 +1,41 @@
+name: assign-class-label tests and linting/formatting
+
+on:
+  push:
+    paths:
+    - container-images/assign-class-label/**
+  pull_request:
+    paths:
+    - container-images/assign-class-label/**
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12'  
+
+      - name: Install dependencies
+        working-directory: ./container-images/assign-class-label/
+        run: |
+          pip install -r requirements.txt
+          pip install -r tests/requirements.txt
```

Comment:
Typically, this file would live in the same directory as `requirements.txt` and would be called `test-requirements.txt`.